File: Exercism Codes/JavaScript/scrabble-score.js
//
// This is only a SKELETON file for the 'Scrabble Score' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const score = (w) => {
  const word = w.toLowerCase();
  let points = 0;
  for (const letter of word) {
    if ("aeioulnrst".includes(letter)) {
      points += 1;
    }
    else if ("dg".includes(letter)) {
      points += 2;
    }
    else if ("bcmp".includes(letter)) {
      points += 3;
    }
    else if ("fhvwy".includes(letter)) {
      points += 4;
    }
    else if ("k".includes(letter)) {
      points += 5;
    }
    else if ("jx".includes(letter)) {
      points += 8;
    }
    else if ("qz".includes(letter)) {
      points += 10;
    }
  }

  return points;
};

File: Exercism Codes/Python/scrabble_score.py
def score(word):
    word_arr = word.lower()
    points = 0
    for letter in word_arr:
        if letter in "aeioulnrst":
            points += 1
        if letter in "dg":
            points += 2
        if letter in "bcmp":
            points += 3
        if letter in "fhvwy":
            points += 4
        if letter in "k":
            points += 5
        if letter in "jx":
            points += 8
        if letter in "qz":
            points += 10

    return points